src/index.twig

```diff
@@ -1,22 +1,15 @@
 {# index.html #}
-{% extends "templates/_pages/page_layout.twig" %}
-
-<head>
-    {% block head %}
-        <link rel="stylesheet" href="css/styles.css"/>
-        <title>{% block title %} Home {% endblock %}</title>
-    {% endblock %}
-</head>
+{% extends "_pages/page_layout.twig" %}
 
 {% block content %}
-
     <h1>Home</h1>
     <div class="posts">
         {% set article = articlesData[0] %}
-        {% include 'templates/_card.twig' with article %}
+        {% include '_card.twig' with article %}
 
         {% set article = blogsData[3] %}
-        {% include 'templates/_card.twig' with article %}
+        {% include '_card.twig' with article %}
```

Comment:
You replaced the template variables in the `_card.twig` template but did not do the adjustment here,
please replace any `with article` with an appropriate object filled with template variables to make sure your template variables are called correctly